crowbar_framework/db/migrate/20150312081459_create_proposals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :barclamp, :null => false
+      t.string :name, :null => false
+      t.text :properties
```

Comment:
Wondering out loud: I wonder if the attributes and deployment parts of the json would be something we should have split here: that is two text columns, one for attributes and one for deployment.

It might make sense as, longer term, most of what is in the deployment shouldn't actually be in the DB imho (only the role assignment). The rest of the deployment bits should actually be a straight copy from the template.